lib/cartopy/tests/mpl/__init__.py

```diff
@@ -270,3 +275,62 @@ def image_as_base64(fname):
 
     html.extend(['</body>', '</html>'])
     return '\n'.join(html)
+
+
+def show(projection, geometry):
+    orig_backend = matplotlib.get_backend()
+    plt.switch_backend('tkagg')
```

Comment:
It doesn't need to change (partially because I'm tempted to remove show altogether in the future), but there shouldn't be a need to force this.